tensorboard/webapp/metrics/views/right_pane/right_pane_test.ts

```diff
@@ -448,34 +448,6 @@ describe('metrics right_pane', () => {
           expect(el.query(By.css('tb-range-input'))).toBeTruthy();
         });
 
-        it('disables tb-range-input on select time disabled', () => {
-          store.overrideSelector(selectors.getIsLinkedTimeEnabled, true);
-          store.overrideSelector(selectors.getMetricsXAxisType, XAxisType.STEP);
-          store.overrideSelector(selectors.getMetricsSelectTimeEnabled, false);
-          store.overrideSelector(selectors.getMetricsUseRangeSelectTime, false);
-          const fixture = TestBed.createComponent(SettingsViewContainer);
-          fixture.detectChanges();
-
-          const el = fixture.debugElement.query(
-            By.css('.linked-time tb-range-input')
-          );
-          expect(el.nativeElement.getAttribute('disabled')).toBe('true');
```

Comment:
You should be able to do:

```
          expect(el.properties.enabled).toBe(false);
```

Reply:
Ah I did not know this. Updated the task. Thanks!